src/lib/netlist/plib/ptests.h

```diff
@@ -15,6 +15,8 @@
 #include <iostream>
 #include <string>
 #include <vector>
+#include <algorithm>
+#include <sstream>
```

Comment:
You might want to get these back into alphabetical order to make it easier to spot duplicates.